content/en/docs/metadata.md

```diff
@@ -40,9 +40,10 @@ There may also be any number of delegated target roles.
 
 Signed by: Root role.
 
-Specifies the other top-level roles. When specifying these roles, the trusted
-keys for each are listed, along with the minimum number of those keys required
-to sign the role's metadata. We call this number the signature threshold.
+The snapshot role signs a metadata file that provides information about the
+latest version of all targets metadata on the repository (the top-level targets
+role and all delegated roles). This information allows clients to know which
+metadata files have been updated and also prevents mix-and-match attacks.
```

Comment:
It looks like you accidentally updated the root role section. 

Also, I don't think we need to completely change the snapshot section. The original text was actually quite okay, apart from the outdated "all but timestamp.json" info. 

Actually, I just saw that there's already a pending fix for this (#120). Would you mind adopting that change into your PR, including @joshuagl's [suggestion](https://github.com/theupdateframework/theupdateframework.io/pull/120#discussion_r1901585808), and adding the file format fix? That would be really appreciated! :)

Reply:
Wow!  Silly mistake! Thanks for noticing I'll update it with @joshuagl  suggestion

Reply:
Done the correction please review @lukpueh